modules/p2/ding_list/ding_list.install

```diff
@@ -54,3 +60,14 @@ function ding_list_update_7001() {
   $mail_settings['ding_list_share'] = "HTMLMailSystem";
   variable_set('mail_system', $mail_settings);
 }
+
+/**
+ * Remove the bookmarks page, if it's been overridden.
+ */
+function ding_list_update_7002() {
+  module_load_include('inc', 'page_manager', 'plugins/tasks/page');
+  $page = page_manager_page_load('user_bookmarks');
+  if (!empty($page)) {
+    page_manager_page_delete($page);
+  }
```

Comment:
I am unsure whether the `ding_list` module is the right place for this update hook since the page being deleted was defined by `ding_user_frontend`. On the other hand this is the module which makes the other page obsolete.

If this comment has not stirred anything then leave this be.